### ISSUE TITLE
MailServer not issuing right the SSL

### ISSUE THREAD

**Issue by franciscopaniskaseker** [litespeedtech/cyberpanel#264]
When I try to issue SSL using "SSL" menu clicking in "MailServer SSL" and then selectin the domain "mail.example.com" I receive as answer:

```
SSL Issued, your mail server now uses Lets Encrypt!
```

but when I test

```bash
openssl s_client -showcerts -connect mail.example.com:993 -servername mail.example.com
```
I get self signed error.

When I try issue SSL through "websites" > "list child domains" > "issue ssl" it never finish and put openlitespeed down sometimes.

**Comment by franciscopaniskaseker** [comment]
Fixed:
I set security of the user to "low" and when I issued SSL from "MailServer SSL" menu it worked.

**Comment by usmannasir** [comment]
Self-signed SSL will be issued if DNS of mail.domain is not pointed to server.

**Comment by franciscopaniskaseker** [comment]
But is pointed since 24 hours ago. I double checked with google dns, cloudflare dns, opendns dns etc.
Also the datacenter recognize the domain to accept new ptr record.

And for some reason with "high" security level the cyberpanel say "ok" for lets encrypt, but actually continue using self sign cert (which was created during offline migration, before change the dns).

When I configured to "low", the same procedure worked (using "SSL" menu clicking in "MailServer SSL" and then selectin the domain "mail.example.com").

**Comment by usmannasir** [comment]
Can you check if mail.domain.com have valid Lets Encrypt cert issued (you can check via browser)

Then do `systemctl restart postfix` `systemctl restart dovecot`

**Comment by franciscopaniskaseker** [comment]
No, it does not has. The SSL only was created after change the security to low.

**Comment by franciscopaniskaseker** [comment]
I will try to reproduce when I install a new fresh server.

**Comment by usmannasir** [comment]
That is why they must have valid SSL. You can issue SSL for them as they are created as child-domains.

**Comment by franciscopaniskaseker** [comment]
Happening with other server.

The user is configured with "low" security. When I try to issue SSL for mail server it return 

```bash
SSL Issued, your mail server now uses Lets Encrypt!
```

But when I check the SSL in 993 port, is a self signed cert.

I tried restart dovecot and postfix, but it did not work.

Also trying to access using https://mail.domain.com also show self signed ssl.

Then I added as child domain and then tried to issue SSL to mail server and it worked.

I saw that DNS module of CyberPanel automatically create "mail." subdomains on DNS Zone. So probably will be good to automatically create a child domain called mail.domain to avoid this problem.
